### PR TITLE
Added map koth:space

### PR DESCRIPTION
**Pull request by JustLetterV**

### CODE REVIEW

**Comment by haykam821** [comment]
This map can be updated to use the `koth:outer_space` dimension type now that dimension type configuration has been added in #88.

**Comment by haykam821** [comment]
The target branch needs to be changed, but otherwise, the map looks good:

Space map

File: src/main/resources/data/koth/games/space.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "type": "koth:koth",
+  "icon": {
+    "id": "minecraft:stick", "Count": 1,
+    "Enchantments": [{"id": "minecraft:knockback", "lvl": 2}]
+  },
+  "short_name": "gameType.koth.koth.short",
+  "description": "gameType.koth.koth.desc",
```
Comment (Restioson) [inline src/main/resources/data/koth/games/space.json:8]:
```suggestion
  "short_name": {
      "translate": "gameType.koth.koth.short"
  },
  "description": {
      "translate": "gameType.koth.koth.desc",
  },
```

File: src/main/resources/data/koth/games/space.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  "type": "koth:koth",
+  "icon": {
+    "id": "minecraft:stick", "Count": 1,
+    "Enchantments": [{"id": "minecraft:knockback", "lvl": 2}]
+  },
+  "short_name": {
+      "translate": "gameType.koth.koth.short"
+  },
+  "description": {
+      "translate": "gameType.koth.koth.desc",
```
[the remark this answers is not in the document]
Comment (Restioson) [reply src/main/resources/data/koth/games/space.json:11]: Hmm, i will fix

**Pull request merged**